exercises/concept/calculator-conundrum/.meta/src/reference/java/CalculatorConundrum.java

```diff
@@ -0,0 +1,30 @@
+public class CalculatorConundrum {
+    private final String answer;
+
+    public CalculatorConundrum(int operand1, int operand2, String operation) {
+        int result;
+        try {
+            if (operation.equals("+")) {
+                result = SimpleOperation.addition(operand1, operand2);
+            } else if (operation.equals("*")) {
+                result = SimpleOperation.multiplication(operand1, operand2);
+            } else if (operation.equals("/")) {
+                result = SimpleOperation.division(operand1, operand2);
+            } else if (operation.equals("")) {
+                throw new IllegalArgumentException("Operation cannot be empty");
+            } else if (operation == null) {
+                throw new NullPointerException("Operation cannot be null.");
```

Comment:
In general, we do not want to throw NullPointerExceptions in Java. This should probably be some new exception created just for this class called `IllegalOperationException` which extends IllegalArgumentException (or just RuntimeException if you prefer). I'd use that for the preceeding and following line as well so that any bad operation throws `IllegalOperationException`.